clients/unshaded/src/main/java/tachyon/client/TachyonFile.java

```diff
@@ -182,7 +186,7 @@ public FileInStream getInStream(ReadType readType) throws IOException {
    */
   public String getLocalFilename(int blockIndex) throws IOException {
     FileBlockInfo blockInfo = getClientBlockInfo(blockIndex);
-    long blockId = blockInfo.getBlockId();
+    long blockId = blockInfo.blockInfo.getBlockId();
```

Comment:
how about change the name of blockInfo to fileBlockInfo? blockInfo.blockInfo doesn't seem to be intuitive.